articles/azure/azs-how-create-load-balancer.md

```diff
@@ -0,0 +1,282 @@
+
+   ![Create a resource option in favourites panel](images/azsp_newmenu.png)
+
+2. In the **New** blade, select **Compute**.
+
+   ![Compute option in New blade](images/azsp_newblade.png)
+
+3. In the **Compute** blade, select the template that you want to use for your VM.
+
+    ![List of VM images in Compute blade](images/azsp_computeblade.png)
+
+4. In the **Create virtual machine** blade, in the **Basics** step, enter general information about the VM, including a name, credentials and resource group, then click **OK**.
+  
+   ![Create virtual machine > Basics](images/azsp_createvm_basics.png)
+
+5. In the *Size* step, select the appropriate size for your VM, depending on its purpose, then click **Select**.
+
+   For information about the different available VM sizes, see <https://docs.microsoft.com/en-gb/azure/azure-stack/user/azure-stack-vm-sizes>
+
+   > [!TIP]
+   > By default, the list shows a selection of recommended VM sizes. To see all available VM sizes, click **View all**.
+
+   ![Create virtual machine > Size](images/azsp_createvm_size.png)
+
+6. In the **Settings** step, change any of the optional settings as required for your VM, then click **OK**. Ensure that the virtual network created earlier is selected and that the VM belongs to an availability set. To create a new availability set:
+
+   - Click **Availability set** on the **Create virtual machine** blade.
+   - Select **Create new** on the **Change availability set** blade.
+   - Enter a name for the availability set and the number of fault domains and update domains, then click **OK**
+
+   ![Create virtual machine > Settings](images/azs-browser-vm-settings-as.png)
+
+7. In the **Summary** step, review the selections you've made and then click **OK** to start the deployment.
+
+8. Create another VM, ensuring that the **Availability set**, **Virtual network**, **Subnet** and **Network Security Group** are the same.
+
+### Create network security group rules
+
+In this section we will create NSG rules to allow inbound traffic.
+
+1. Navigate to your network security group by clicking on **All services** in the favourites panel, then selecting **Network security groups** under the networking section.
+
+2. Select the network security group your VMs are using from the list.
+
+3. In the blade for your network security group, select **Inbound security rules** under the **Settings** section.
+
+4. Click the **+Add** button.
+
+5. In the **Add inbound security rule** blade, enter the following information:
+
+   - **Source** - The source filter can be Any, an IP address range, or a default tag. It specifies the incoming traffic from a specific source IP address range that will be allowed or denied by this rule.
+
+   - **Source port range** - Provide a single port, such as 80, or a port range, such as 1024-65535. This specifies from which ports traffic will be allowed or denied by this rule. Provide an asterisk (*) to allow traffic on any port.

```

Comment:
Use an asterisk (*) to allow traffic on any port.

Also, you may have to escape the asterisk using a backslash (\) to prevent any funky italics

Reply:
Fixed